.github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Preguntar si el software está dentro del alcance de rOpenSci
+about: ¿Quieres confirmar que tu paquete encaja para revisión? Usa esta plantilla para obtener repuesta de les editores en Español (Experimental)
```

Comment:
In one other document the approach to handling gender-biased language was to try use a neutral form or a feminine -- not a masculine. If we are consistent here, "les editores" would become "las personas que editan" or "las editoras".